torch/csrc/distributed/c10d/init.cpp

```diff
@@ -743,6 +746,14 @@ They are used in specifying strategies for reduction collectives, e.g.,
       },
       py::call_guard<py::gil_scoped_release>());
 
+  module.def("_default_first_bucket_bytes", []() {
+    return ::c10d::DEFAULT_FIRST_BUCKET_BYTES;
+  });
+
+  module.def("_default_broadcast_bucket_bytes", []() {
+    return ::c10d::DEFAULT_BROADCAST_BUCKET_BYTES;
+  });
```

Comment:
Does these two have to be methods? Can we use `module.attr` instead? E.g.

```c++
  module.attr("_DEFAULT_FIRST_BUCKET_BYTES") =
      py::cast(kDefaultBroadcastBucketBytes);
```

For consistency, we use capital letters in constants, and use `kCamelName` in C++ to name it.  One example is `_DEFAULT_NUM_SEND_RECV_THREADS` in `rpc/init.cpp`